src/modules/users/services/PermissionsService.ts

```diff
@@ -0,0 +1,29 @@
+// Not a real example of a service, do not use for non-testing purposes
+export class PermissionsService {
```

Comment:
this is ugly as hell, but I ran out of time, and this part of the code irrelevant for what this PR is aiming to achieve.